mmcv/cnn/bricks/conv_module.py

```diff
@@ -147,6 +145,11 @@ def __init__(self,
                 norm_channels = in_channels
             self.norm_name, norm = build_norm_layer(norm_cfg, norm_channels)
             self.add_module(self.norm_name, norm)
+            if self.with_bias:
```

Comment:
May add a comment here to indicate why we only report warning for BN/IN here.

Reply:
How about changing the warning message into "'Unnecessary conv bias before batch/instance norm'"